WebDevStudios/wd_s
# Bulk Dependabot for 06-02-2021

Closes #639
Closes #641
Closes #658
Closes #661

---

### Description

@coreymcollins This should get us all caught up! I've tested locally and everything appears to build/compile/watch correctly. Please give this PR your 🦅 👁️ 

The following Node dependencies have been updated:

```bash
 @arkweid/lefthook                                              ^0.7.2  →   ^0.7.6     
 @tailwindcss/postcss7-compat                                   ^2.1.1  →   ^2.1.4     
 @wordpress/prettier-config                                     ^1.0.3  →   ^1.0.5     
 @wordpress/scripts                                            ^14.1.1  →  ^16.1.1     
 css-loader                                                     ^5.2.4  →   ^5.2.6     
 eslint-webpack-plugin                                          ^2.5.3  →   ^2.5.4     
 mini-css-extract-plugin                                        ^0.9.0  →   ^1.6.0     
 tailwindcss                   npm:@tailwindcss/postcss7-compat@^2.1.1  →   ^2.1.4    
```

In addition, this PR also updates the following PHP dependencies:

```bash
composer/installers   ^0.7.0 → ^1.11.0
wpcli/wpcli           ^* →     ^2.5.0
```

---

## Other

### @wordpress/scripts v16+ Info

With v16+ support for IE11 has been dropped. See the [CHANGELOG](https://github.com/WordPress/gutenberg/blob/HEAD/packages/scripts/CHANGELOG.md)

### Webpack 5 Info

The following dependencies _must_ remain "locked into" the versions currently listed in `package.json`. **Anything newer is for Webpack 5 and will break @wordpress/scripts and our build!**

- css-minimizer-webpack-plugin `1.3.0`
- copy-webpack-plugin `6.4.1`
-  postcss-loader  `4.2.0`      
- sass-loader `10.2.0`
- svg-spritemap-webpack-plugin    `3.9.1`

### Finally...

The PR's listed below need to have the following comment added (to ignore future @dependabot updates):

> @dependabot ignore this major version

- #662
- #664
- #657

---

### Screenshots

Successful Build:

![screenshot](https://dl.dropbox.com/s/kqlxft7h40zqetr/Screen%20Shot%202021-06-02%20at%2015.47.15.png?dl=0)

---

### STEPS TO VERIFY

1. `gh pr checkout 665`
2. `npm ci`
3. `npm run build`
4. `npm run watch` and verify Webpack recompiles.
